test/e2e/storage/testsuites/base.go

```diff
@@ -421,6 +423,45 @@ func convertTestConfig(in *PerTestConfig) volume.TestConfig {
 	}
 }
 
+// getSizeRangesIntersection takes two instances of storage size ranges and determines the
+// intersection of the intervals (if it exists) and return the minimum of the intersection
+// to be used as the claim size for the test.
+func getSizeRangesIntersection(first volume.SizeRange, second volume.SizeRange) string {
+	var firstMin, firstMax, secondMin, secondMax resource.Quantity
+	var err error
+	if firstMin, err = resource.ParseQuantity(first.Min); err != nil {
+		framework.Failf("Failed parsing quantity %s", first.Min)
+	}
+
+	if firstMax, err = resource.ParseQuantity(first.Max); err != nil {
+		framework.Failf("Failed parsing quantity %s", first.Max)
+	}
+
+	if secondMin, err = resource.ParseQuantity(second.Min); err != nil {
+		framework.Failf("Failed parsing quantity %s", second.Min)
+	}
+
+	if secondMax, err = resource.ParseQuantity(second.Max); err != nil {
+		framework.Failf("Failed parsing quantity %s", second.Max)
+	}
+
+	if secondMin.Cmp(firstMax) > 0 || firstMin.Cmp(secondMax) > 0 {
+		framework.Failf("Intersection of size ranges %+v, %+v is null", first, second)
```

Comment:
It's not an error if there is no intersection. Instead the test needs to be skipped because the test suite is incompatible with the driver.

Reply:
change to `framework.Skipf(xxx)` will auto skip current testing , right?